smoke-test/tests/cypress/cypress/e2e/glossary/glossary_navigation.js

```diff
@@ -30,6 +30,15 @@ describe("glossary sidebar navigation test", () => {
         cy.get('[data-testid="glossary-browser-sidebar"]').contains(glossaryTermGroup).click().wait(3000);
         cy.get('*[class^="GlossaryEntitiesList"]').contains(glossaryTerm).should("be.visible");
 
+        // create glossaryParentGroup
+        cy.goToGlossaryList();
```

Comment:
Can you explain why this is necessary? Seems that glossary_navigation is still not passing : /